test/box/update.result

```diff
@@ -1695,11 +1695,20 @@ t:upsert(ops)
 - [1, {}, [], [1, 2, 3, [2, 5, 6, 7, [8, 9, [10, 11, 12], 13, 14], 15, 16, [17, 18,
         19]], 20, {'b': 22, 'a': 21, 'c': {'d': 23, 'e': 24}}, 25]]
 ...
--- First two operations produce zero length bar update in field
--- [4][4][5][4]. The third operation should raise an error.
--- ops = {{'+', '[4][4][5][4]', 13000}, {'=', '[4][4][5][1]', 8000}, {'+', '[4][4][5][4]', 1}}
--- t:update(ops)
--- t:upsert(ops)
+-- First two operations produce zero length bar update in field [4][4][5][4].
```

Comment:
Ok, I see now why you commented out the tests in the prev patch. It is not a problem to change the same test multiple times during the patchset if it is justified. Lets not comment anything out.

Reply:
AFAIR at least some of the tests leads to assertion. I'll then comment only such tests.

Reply:
Adjusted patch set a bit to avoid temporary assertions and as a result avoid commenting tests out.